vitest.config.ts

```diff
@@ -7,6 +7,9 @@ export default mergeConfig(
   defineConfig({
     test: {
       environment: 'jsdom',
+      snapshotFormat: {
+        printBasicPrototype: true
+      },
```

Comment:
Only jest `<29.0.0` uses this snapshot configuration. Modern versions of `jest` and `vitest` avoid using it this way.
```suggestion
```